src/viewer/reference/state.md

```diff
@@ -167,51 +261,31 @@ Annotation related fields and methods:
 | `removeAnnotation(annotation, options)` | Remove the given annotation from state                          |
 | `clearAnnotations()`                    | Remove all annotations from state                               |
 
-## Hub
-
-`state.hub` allows to listen for [state update events](#events) :
-
-```javascript
-state.hub.on("objects-selected", ({ ids, options }) => {
-  console.log("Do something.");
-});
-
-state.hub.on(
```

Comment:
The usage examples were good. Are they still here and I missed them or they've been removed?

Reply:
I removed them, I'll put them back.